lib/Common/CommonDefines.h

```diff
@@ -325,7 +325,7 @@
 
 ////////
 //Time Travel flags
-#define ENABLE_TTD 0
+#define ENABLE_TTD 1
```

Comment:
Yeah, so I turned it off during the merge to get the merge to build cleanly- and then turned it back on to fix up TTD specific conflicts.